website/docs/ReactNavigation.md

```diff
@@ -198,6 +202,155 @@ describe('Testing react navigation', () => {
 });
 ```
 
+## Drawer Navigator
+
+Testing the Drawer Navigation requires an additional setup step for mocking the Reanimated library.
+
+### Setting up
+
+Install the packages required for React Navigation. For this example, we will use a [drawer navigator](https://reactnavigation.org/docs/drawer-navigator/) to transition between a home screen and an additional screen.
+
+```
+$ yarn add @react-native-community/masked-view @react-navigation/native @react-navigation/drawer react-native-gesture-handler react-native-reanimated react-native-safe-area-context react-native-screens
```

Comment:
```suggestion
$ yarn add @react-navigation/native @react-navigation/drawer react-native-gesture-handler react-native-reanimated react-native-safe-area-context react-native-screens
```

Reply:
@satya164 You are suggesting the `masked-view` is not needed? It's listed in the install instructions on the React Navigation website: https://reactnavigation.org/docs/getting-started#installing-dependencies-into-a-bare-react-native-project

Reply:
It's only necessary for stack navigator.

Reply:
Since this PR would create one set of instructions for both the stack navigator and the drawer navigator - should I just leave the masked view here? Or add instructions for the stack navigator only?

Reply:
Sounds good to me